mypy/messages.py

```diff
@@ -992,6 +998,31 @@ def pretty_or(args: List[str]) -> str:
     return ", ".join(quoted[:-1]) + ", or " + quoted[-1]
 
 
+def append_invariance_notes(notes: List[str], arg_type: Instance, expected_type: Instance) -> List[str]:
```

Comment:
@ilevkivskyi Sorry, this is an easier way to make this comment. What is the preferred way to move part of this definition to the next line?